### ISSUE TITLE
Request: Create web pages for cWB statistics only

### ISSUE THREAD

**Issue by AmberStuver** [gwpy/vet#26]
This is a request:

Running VET over cWB triggers has been broken for a long time due to plotting errors that occur after the basic statistics are calculated.  Brennan and I have been copy & pasting the statistics that VET reports to the terminal.  However, it would be nice if we can temporarily disable the plotting so that the VET pages are created with the statistics so that there is something more official than our copy/paste to share with each other, and the collaboration.

**tl;dr - until VET is fixed for cWB, please make cWB VET pages without plots so that we can have the statistics documentation.**

**Comment by AmberStuver** [comment]
I don't think this is an issue anymore.
